sys/sched_round_robin/sched_round_robin.c

```diff
@@ -0,0 +1,110 @@
+/*
+ * Copyright (C) 2021 TUBA Freiberg
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+/**
+ * @ingroup     sys
+ * @{
+ *
+ * @file
+ * @brief       Round Robin Scheduler implementation
+ *
+ * @author      Karl Fessel <karl.fessel@ovgu.de>
+ *
+ * @}
+ */
+
+#include "sched.h"
+#include "thread.h"
+#include "xtimer.h"
+#include "sched_round_robin.h"
+
+#define ENABLE_DEBUG 0
+#include "debug.h"
+
+static void sched_round_robin_cb(void *d);
+
+static xtimer_t rr_timer = { .callback = sched_round_robin_cb };
```

Comment:
please prefix statics with "_".

Reply:
are you sure?
I always thought of _function as beeing libc internals, names that your libc provider can take at any time.
sys/stdio_rtt/stdio_rtt.c has a buffer `static char up_buffer   [STDIO_TX_BUFSIZE];`

Reply:
did that